pallets/stream/src/lib.rs

```diff
@@ -328,10 +356,31 @@ pub mod pallet {
 				ensure!(stream_details.registry == registry_id, Error::<T>::UnauthorizedOperation);
 			}
 
+			let registry_id =
```

Comment:
Identifier creation is only required at the stream creation level. We don't need it here.